x-pack/plugins/siem/public/components/open_timeline/helpers.test.ts

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { cloneDeep, omit } from 'lodash/fp';
+
+import { getNotesCount, getPinnedEventCount, isUntitled } from './helpers';
+import { mockTimelineResults } from '../../mock/timeline_results';
+import { TimelineResult } from './types';
+
+describe('helpers', () => {
+  let mockResults: TimelineResult[];
+
+  beforeEach(() => {
+    mockResults = cloneDeep(mockTimelineResults);
+  });
+
+  describe('#getPinnedEventCount', () => {
+    test('returns 6 when the timeline has 6 pinned events', () => {
+      const with6Events = mockResults[0];
+
+      expect(getPinnedEventCount(with6Events)).toEqual(6);
+    });
+
+    test('returns zero when the timeline has an empty collection of pinned events', () => {
+      const withPinnedEvents = { ...mockResults[0], pinnedEventIds: {} };
+
+      expect(getPinnedEventCount(withPinnedEvents)).toEqual(0);
+    });
+
+    test('returns zero when pinnedEventIds is undefined', () => {
+      const withPinnedEvents = omit('pinnedEventIds', { ...mockResults[0] });
```

Comment:
`omit` is deprecated and removed in 5, fyi:
https://github.com/lodash/lodash/issues/2930

Reply:
with the merge to `master` happening any moment now, let's remove all `omit`s here and in the rest of the project